docs/docs/gatsby-on-linux.md

```diff
@@ -64,10 +62,11 @@ Or to install all at the same time and approve `(y)` all installs:
 sudo apt update && sudo apt -y upgrade && sudo apt install build-essential && sudo apt install git && sudo apt install libpng-dev
 ```
 
-<!-- links -->
+### Additional links and resources
 
-[windows store]: https://www.microsoft.com/en-us/store/p/ubuntu/9nblggh4msv6
-[n]: https://github.com/tj/n
-[n-install]: https://github.com/mklement0/n-install
-[nvm]: https://github.com/creationix/nvm
-[bash startup]: https://github.com/Microsoft/WSL/issues/776#issuecomment-266112578
+- [Super detailed guide to making VSCode work with ESL from VSCode's docs website](https://code.visualstudio.com/docs/remote/wsl)
+- [windows store](https://www.microsoft.com/en-us/store/p/ubuntu/9nblggh4msv6)
+- [n](https://github.com/tj/n)
+- [n-install](https://github.com/mklement0/n-install)
+- [nvm](https://github.com/creationix/nvm)
+- [bash startup](https://github.com/Microsoft/WSL/issues/776#issuecomment-266112578)
```

Comment:
One line description would be helpful before visiting. As the reader can pick what is useful or not for his/her context.

Reply:
Hmmm since they are all mentioned in the article, I could explain them there. I just enabled the links!